view/adminhtml/templates/indexing-manager/overview.phtml

```diff
@@ -0,0 +1,14 @@
+<?php
+/** @var \Magento\Backend\Block\Template $block */
+?>
+<div class="algolia-indexing-manager-overview">
+    <div class="algolia_dashboard_warning">
+        <p>Here is a list of all the indices created by the extension on the Algolia Dashboard.</p>
```

Comment:
I really love this new feature - but we should clarify that the indices listed exclude replicas.

Reply:
Done!

I think there is a lot of opportunities to improve the page here (thinking of displaying the number of replicas, number of records, the application each index belong to, last update, etc ... ) . Let's improve this all along the indexing improvement phases !